packages/core/core.js

```diff
@@ -1,35 +1,38 @@
-import configureLogger from 'meteor/unchained:core-logger';
-import configureDelivery from 'meteor/unchained:core-delivery';
-import configurePayment from 'meteor/unchained:core-payment';
-import configureWarehousing from 'meteor/unchained:core-warehousing';
-import configureProducts from 'meteor/unchained:core-products';
-import configureBookmarks, {
-  services as bookmarkServices,
-} from 'meteor/unchained:core-bookmarks';
-import configureQuotations from 'meteor/unchained:core-quotations';
-import configureCurrencies from 'meteor/unchained:core-currencies';
-import configureCountries from 'meteor/unchained:core-countries';
-import configureLanguages from 'meteor/unchained:core-languages';
-import configureDocuments from 'meteor/unchained:core-documents';
-import configureOrders from 'meteor/unchained:core-orders';
-import configureAssortments from 'meteor/unchained:core-assortments';
-import configureFilters from 'meteor/unchained:core-filters';
-import configureEnrollments from 'meteor/unchained:core-enrollments';
-import configureWorker from 'meteor/unchained:core-worker';
-import configureMessaging from 'meteor/unchained:core-messaging';
-import configureEvents from 'meteor/unchained:core-events';
+import { db } from 'meteor/unchained:core-mongodb';
 
-export default async ({
+// import configureUsers from 'meteor/unchained:core-users';
+// import configureAccounts from 'meteor/unchained:core-accountsjs';
+// import configureDelivery from 'meteor/unchained:core-delivery';
+// import configurePayment from 'meteor/unchained:core-payment';
+// import configureWarehousing from 'meteor/unchained:core-warehousing';
+// import configureProducts from 'meteor/unchained:core-products';
+// import configureQuotations from 'meteor/unchained:core-quotations';
+// import configureCurrencies from 'meteor/unchained:core-currencies';
+// import configureCountries from 'meteor/unchained:core-countries';
+// import configureLanguages from 'meteor/unchained:core-languages';
+// import configureDocuments from 'meteor/unchained:core-documents';
+// import configureOrders from 'meteor/unchained:core-orders';
+// import configureAssortments from 'meteor/unchained:core-assortments';
+// import configureFilters from 'meteor/unchained:core-filters';
+// import configureEnrollments from 'meteor/unchained:core-enrollments';
+// import configureWorker from 'meteor/unchained:core-worker';
+// import configureMessaging from 'meteor/unchained:core-messaging';
+
+import { configureLogs } from 'meteor/unchained:core-logger';
+import { configureEvents } from 'meteor/unchained:core-events';
+import { configureBookmarks } from 'meteor/unchained:core-bookmarks';
+
+// import { bookmarkServices } from 'meteor/unchained:bookmark-services';
+
+export const initCore = async ({
   modules = {},
   migrationRepository,
   ...otherComponents
 } = {}) => {
   const moduleOptions = {
     migrationRepository,
   };
-  configureLogger(modules.logger, moduleOptions);
```

Comment:
I think the db connection should open explicitly here and not at "import { db } from 'meteor/unchained:core-mongodb';" so it should be more of

```
import { connect } from 'meteor/unchained:core-mongodb';

initCor = () {
...
   const db = connect(MONGO_URL, mongoDBOptions);

   configureBla({db, ...rest});
}
```